src/objects/zcl_abapgit_objects_super.clas.abap

```diff
@@ -313,6 +313,7 @@ CLASS ZCL_ABAPGIT_OBJECTS_SUPER IMPLEMENTATION.
         suppress_protocol         = abap_false
         with_program_includes     = abap_false
         suppress_dictionary_check = abap_false
+        phased_activation         = abap_false
```

Comment:
suggest keeping out this change, but this should not change anything, it just adds the parameter default to the code